ros_ws/src/autonomous/wallfollowing2/script/wallfollowing.py

```diff
@@ -17,17 +17,21 @@
 TOPIC_DRIVE_PARAMETERS = "/input/drive_param/autonomous"
 TOPIC_LASER_SCAN = "/scan"
 
-SLOW = 0.15
+SLOW = 0.2
 FAST = 1.0
 
-RADIUS_SMALL = 2
-RADIUS_BIG = 10
+RADIUS_SMALL = 0
+RADIUS_BIG = 30
 
-ERROR_SPEED_DECREASE = 1.7
-ERROR_DEAD_ZONE = 0.2
+ERROR_SPEED_DECREASE = 4
+ERROR_DEAD_ZONE = 0.15
 
 UPDATE_FREQUENCY = 60
 
+MAX_ACCELERATION = 0.5
+
+last_speed = 0
+
```

Comment:
If those values are not constants (as they are changed) could they be move to the `.launch` file as args?